src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -65,33 +74,72 @@ protected override bool ValidateParameters()
         {
             _createPDBCommand = CompilationEntry.GetMetadata(MetadataKeys.CreatePDBCommand);
 
-            if (CrossgenTool == null && Crossgen2Tool == null)
-            {
-                return false;
-            }
             if (IsPdbCompilation && CrossgenTool == null)
             {
-                // We need the crossgen tool for now to emit native symbols. Crossgen2 does not yet support this feature
+                // PDB compilation is a step specific to Crossgen1 and 5.0 Crossgen2
```

Comment:
What happens if we are using Crossgen2 version ≥ 6 and `IsPdbCompilation` is `true`?  That seems a possible combination unless we also change code in PrepareForReadyToRunCompilation.cs.

Reply:
I'll fix this in PrepareReadyToRunCompilation, Crossgen2 is currently not capable of lazy PDB / PerfMap generation (R2RDump currently supports that) and the scripts don't actually need that.

Reply:
Hopefully fixed in 6th commit.